frontend/www/js/omegaup/user/profile.ts

```diff
@@ -6,12 +6,17 @@ import * as api from '../api';
 import * as ui from '../ui';
 import T from '../lang';
 
-// TODO: Import Profile.vue when PR #5951 is merged
-import user_Profile from '../components/user/Profilev2.vue';
+import user_Profile from '../components/user/Profile.vue';
 
 OmegaUp.on('ready', () => {
   const payload = types.payloadParsers.UserProfileDetailsPayload();
   const locationHash = window.location.hash.substr(1).split('#');
+  let selectedTab = locationHash[0] !== '' ? locationHash[0] : 'view-profile';
```

Comment:
```suggestion
  let selectedTab = locationHash[0] || 'view-profile';
```

hay que intentar evitar los ternarios cuando haya alternativas.